cmake/build_ace_tao.cmake

```diff
@@ -21,6 +21,9 @@ set(_opendds_std "$<TARGET_PROPERTY:${_opendds_std_target},CXX_STANDARD>")
 find_package(Perl REQUIRED)
 if(OPENDDS_STATIC)
   list(APPEND _OPENDDS_CONFIGURE_ACE_TAO_ARGS --static=1)
+  if(NOT DEFINED CMAKE_MSVC_RUNTIME_LIBRARY OR CMAKE_MSVC_RUNTIME_LIBRARY MATCHES "DLL")
```

Comment:
If we're accounting for `CMAKE_MSVC_RUNTIME_LIBRARY` here, we should also probably do it in `_opendds_vs_force_static`.

Reply:
This was made assuming the `OPENDDS_STATIC_RUNTIME` change was unintentional. Since it's not, this code is mutually exclusive with the `_opendds_vs_force_static`.